File: frameworks/Java/fair/java-kora2-jdbc-repository/src/main/java/io/techempower/benchmark/kora/model/World.java
package io.techempower.benchmark.kora.model;

import io.koraframework.database.common.annotation.Column;
import io.koraframework.database.common.annotation.Id;
import io.koraframework.database.jdbc.annotation.EntityJdbc;
import io.koraframework.json.common.annotation.Json;

@Json
@EntityJdbc
public record World(@Id @Column("id") int id,
                    @Column("randomnumber") int randomNumber) {
}

File: frameworks/Java/fair/kotlin-kora2-jdbc-repository/src/main/kotlin/io/techempower/benchmark/kora/model/World.kt
package io.techempower.benchmark.kora.model

import io.koraframework.database.common.annotation.Column
import io.koraframework.database.common.annotation.Id
import io.koraframework.database.jdbc.annotation.EntityJdbc
import io.koraframework.json.common.annotation.Json

@Json
@EntityJdbc
data class World(
    @field:Id @field:Column("id") val id: Int,
    @field:Column("randomnumber") val randomNumber: Int,
)
